session/filesync/diffcopy.go

```diff
@@ -82,6 +82,12 @@ func recvDiffCopy(ds grpc.ClientStream, dest string, cu CacheUpdater, progress p
 		cf = cu.HandleChange
 		ch = cu.ContentHasher()
 	}
+	defer func() {
```

Comment:
Is it worth combining the two defers in this function?

Reply:
I hope this can be removed with upstream change so better to keep it separate block with a comment

Reply:
That's fair